src/System.EventLog/src/System/Diagnostics/SafeEventLogReadHandle.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeEventLogReadHandle : SafeHandleZeroOrMinusOneIsInvalid
+    {
+        internal SafeEventLogReadHandle() : base(true) { }
+
+#pragma warning disable BCL0015�// Disable Pinvoke analyzer errors.
```

Comment:
Instead of BCL0015, please create a baseline file `src\System.EventLog\src\PinvokeAnalyzerExceptionList.analyzerdata.netcoreapp`
and in it put these one per line like
````
advapi32.dll!OpenEventLog
advapi32.dll!CloseEventLog
```

Here's an  example:
https://github.com/dotnet/corefx/blob/master/src/System.Console/src/PinvokeAnalyzerExceptionList.analyzerdata.netcoreapp

I know I said I didn't care much about it, but today I'm dealing with the fallout of missing one of these in a different context, so I would like to be more rigorous about using baseline files instead (those are harder to overlook)

Reply:
completed